controllers/articleRatingController.js

```diff
@@ -0,0 +1,63 @@
+import models from '../models';
+import Response from '../helpers/responseHelper';
+import { STATUS } from '../helpers/constants';
+import getAverageRatings from '../helpers/articleRatingHelper';
+
+const { ratings } = models;
+/**
+ * Class representing article rating controller.
+ *
+ * @export
+ * @class ArticleRatingController
+*/
+export default class ArticleRatingController {
+  /**
+   * Sends the request payload to the database and returns the article object
```

Comment:
#### Nitpick
- Not descriptive enough

Reply:
@aanchirinah Thank you for pointing that out, I will add more description